scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -107,6 +107,15 @@ class ScoverageInstrumentationComponent(val global: Global, extraAfterPhase: Opt
   private var options: ScoverageOptions = new ScoverageOptions()
   private var coverageFilter: CoverageFilter = AllCoverageFilter
 
+  private val isScalaJsEnabled: Boolean = {
+    try {
+      getClass.getClassLoader.loadClass("scala.scalajs.js.Any")
```

Comment:
This is testing whether `scala.scalajs.js.Any` is on the classpath *of the compiler*. This is not what you want. You want to test whether it is on the classpath *of the project being compiled*. You can test that with
```scala
rootMirror.getClassIfDefined("foo") != NoSymbol
```

Reply:
Addressed in https://github.com/scoverage/scalac-scoverage-plugin/pull/281/commits/be4e95d50e44b26c9d189d1105b17bcf2f0d10a5 and https://github.com/scoverage/scalac-scoverage-plugin/pull/281/commits/3c76241db11f5ab708233302cf704360ba18ad88